Index: sogo-connector.tb31-sogo-connector-31.0.5/install.rdf
===================================================================
--- sogo-connector.tb31-sogo-connector-31.0.5.orig/install.rdf
+++ sogo-connector.tb31-sogo-connector-31.0.5/install.rdf
@@ -5,7 +5,7 @@
   <Description about="urn:mozilla:install-manifest"
     em:name="Inverse SOGo Connector"
     em:id="sogo-connector@inverse.ca"
-    em:version="31.0.4"
+    em:version="31.0.5"
     em:description="A DAV plugin for keeping addressbooks and events in sync"
     em:iconURL="chrome://sogo-connector/skin/addressbook/sogo.png"
     em:creator="Inverse &lt;support@inverse.ca>"
