Index: sogo-connector.tb24-sogo-connector-24.0.6/Makefile
===================================================================
--- sogo-connector.tb24-sogo-connector-24.0.6.orig/Makefile
+++ sogo-connector.tb24-sogo-connector-24.0.6/Makefile
@@ -1,5 +1,4 @@
 PACKAGE = sogo-connector
-GIT_REV = $(shell git rev-parse --verify HEAD | cut -c1-10)
 
 ifeq ($(shell uname),Darwin)
 VERSION = $(shell grep em:version install.rdf | sed -E 's@(em:version=|"| )@@g')
@@ -7,7 +6,7 @@ else
 VERSION = $(shell grep em:version install.rdf | sed -e 's@\(em:version=\|\"\|\ \)@@g')
 endif
 
-XPI_ARCHIVE = $(PACKAGE)-$(VERSION)-$(GIT_REV).xpi
+XPI_ARCHIVE = $(PACKAGE)-$(VERSION).xpi
 
 SHELL = /bin/bash
 ZIP = /usr/bin/zip
@@ -42,8 +41,8 @@ MANIFEST-pre:
 	@find . -type f -name "*.properties" >> $@	
 	@find . -type f -name "RELEASE-NOTES" >> $@	
 
-rest:
-	@make $(XPI_ARCHIVE)
+rest: MANIFEST
+	@+$(MAKE) $(XPI_ARCHIVE)
 
 $(XPI_ARCHIVE): $(FILENAMES)
 	@echo Generating $(XPI_ARCHIVE)...
